### ISSUE TITLE
Dynamically find enchantments

### ISSUE THREAD

**Issue by amot-dev** [amot-dev/cursed-table-revamped#10]
A way to dynamically find all available enchantments, and their current status in the enchanting table. This would allow modded enchants to appear and be toggle-able.

**Comment by Det-rovv** [comment]
So right now your mod adding to enchantment table only vanilla curses?

**Comment by amot-dev** [comment]
Yeah, since it's done with gamerules. Eventually I want to have it be controlled by data pack, which would allow modded enchants more easily.

**Comment by amot-dev** [comment]
So turns out Minecraft already handles enchantment replacement in the table via the `#minecraft:in_enchanting_table` tag. You can create a data pack.

You can add to this file like so:
```json
{
  "replace": false,
  "values": [
    "minecraft:binding_curse",
    "minecraft:vanishing_curse",
    "somemod:somecurse"
  ]
}

```

I'll keep Cursed Table Revamped in its current state for ease of use via Gamerules.
